base_phone/wizard/number_not_found_view.xml

```diff
@@ -39,7 +39,7 @@
                         options="{'dial_button_invisible': True}"/>
                     <field name="current_partner_mobile" widget="phone"
                         options="{'dial_button_invisible': True}"/>
-                    <button name="update_partner" icon="gtk-convert" colspan="2"
+                    <button name="update_partner" class="oe_highlight" colspan="2"
                         string="Update Partner with this Number" type="object"/>
```

Comment:
@yvaucher your proposal is certainly a good idea. We'll try to do that in another PR.